devicemaps.json

```diff
@@ -704,6 +832,1363 @@
+            "pciAddress": "0000:04:00.1",
+            "type": "WAN",
+            "speed": 1000,
+            "name": "ge-2",
+            "description": "Port 2 labeled on the device",
+            "bcpNetwork": {
+              "standaloneBranch": {
+                "name": "wan3",
+                "description": "WAN 3 network interface, connected to port 2 labeled on the device"
+              }
+            }
+          },
+          {
+            "pciAddress": "0000:04:00.0",
+            "type": "LAN",
+            "speed": 1000,
+            "name": "ge-3",
+            "description": "Port 3 labeled on the device",
+            "bcpNetwork": {
+              "standaloneBranch": {
+                "name": "",
+                "description": ""
+              }
+            }
+          },
+          {
+            "pciAddress": "0000:03:00.1",
+            "type": "LAN",
+            "speed": 1000,
+            "name": "ge-4",
+            "description": "Port 4 labeled on the device",
+            "bcpNetwork": {
+              "standaloneBranch": {
+                "name": "",
+                "description": ""
+              }
+            }
+          },
+          {
+            "pciAddress": "0000:03:00.0",
+            "type": "LAN",
+            "speed": 1000,
+            "name": "ge-5",
+            "description": "Port 5 labeled on the device",
+            "bcpNetwork": {
+              "standaloneBranch": {
+                "name": "lan",
+                "description": "LAN network interface, connected to port 5 labeled on the device"
+              }
+            }
+          },
+          {
+            "pciAddress": "0000:02:00.1",
+            "type": "HAFabric",
+            "speed": 1000,
+            "name": "ge-6",
+            "description": "Port 6 labeled on the device",
+            "bcpNetwork": {
+              "standaloneBranch": {
+                "name": "ha-fabric",
+                "description": "HA fabric network interface, connected to port 6 labeled on the device"
+              }
+            }
+          },
+          {
+            "pciAddress": "0000:02:00.0",
+            "type": "HASync",
+            "speed": 1000,
+            "name": "ge-7",
+            "description": "Port 7 labeled on the device",
+            "bcpNetwork": {
+              "standaloneBranch": {
+                "name": "ha-sync",
+                "description": "HA sync network interface, connected to port 7 labeled on the device"
+              }
+            }
+          }
+        ],
+        "lte": [
+          {
+            "targetInterface": "wwp0s21u1i8",
+            "type": "WAN",
+            "name": "lte-0",
+            "description": "LTE device interface",
+            "bcpNetwork": {
+              "standaloneBranch": {
+                "name": "lte-0",
+                "description": "LTE network interface"
+              }
+            }
+          }
+        ]
+      },
+      "SSR1200": {
```

Comment:
@BenMatase for now lets just remove all the SSR1xxx and let Avi and Reid provide the updates there.